README.Rmd

```diff
@@ -42,73 +42,67 @@ knitr::opts_chunk$set(
 [![rstudio mirror downloads](https://cranlogs.r-pkg.org/badges/rredlist)](https://github.com/metacran/cranlogs.app)
 [![cran version](https://www.r-pkg.org/badges/version/rredlist)](https://cran.r-project.org/package=rredlist)
 
-`rredlist` is an R client for the IUCN Red List (http://apiv3.iucnredlist.org/api/v3/docs). 
+`rredlist` is an R client for the IUCN Red List (<https://apiv3.iucnredlist.org/api/v3/docs>). 
 The IUCN Red List is a global list of threatened and endangered species.
 
-IUCN Red List docs: http://apiv3.iucnredlist.org/api/v3/docs
+[IUCN Red List docs][docs]
 
-> important note: [redlistr][] is a different package - not working with the IUCN Red List API
+> What rredlist is not: [redlistr][] is a different package - not working with the IUCN Red List API; Furthermore, rredlist does not include support for the spatial API, described at
+<https://apiv3.iucnredlist.org/spatial>.
 
 > Note: there can sometimes be a discrepancy between what you get on the IUCN website and what
 > you get with this package; we don't know why, the IUCN API is not an open book.
 
-## Citing the IUCN Red List API
-
-See http://apiv3.iucnredlist.org/about
-
-## Authentication
```

Comment:
you can't use the package without authentication though

Reply:
so if they run through an example first they will just get errors

Reply:
Right, but when they first scroll they want to see what they can do? I added a sentence at the beginning, would that be a compromise?

Reply:
sounds good